lib/miq_automation_engine/models/miq_ae_domain.rb

```diff
@@ -117,15 +121,19 @@ def about_file_name
     File.join(MiqAeDatastore::DATASTORE_DIRECTORY, "#{about.fqname}#{CLASS_DIR_SUFFIX}", CLASS_YAML_FILENAME) if about
   end
 
-  def self.import_options(git_repo, ref, ref_type)
-    options = {'git_dir' => git_repo.directory_name, 'preview' => false}
-    case ref_type
+  def self.import_options(git_repo, options)
+    options['git_dir'] = git_repo.directory_name
+    options['preview'] ||= false
+    options['ref'] ||= DEFAULT_BRANCH
+    options['ref_type'] ||= BRANCH
+    options['ref_type'] = options['ref_type'].downcase
```

Comment:
Minor, but align the `=`...no need to change in this PR, unless you end up with more commits.